README.md

```diff
@@ -214,6 +220,27 @@ The [SolaceProtoBuffRecordTest](solace-apache-beam-samples/src/main/java/com/sol
     ```
 1. Verify that the messages are being outputted in the log.
 
+#### SolaceBigQuery
+
+The [SolaceBigQuery](solace-apache-beam-samples/src/main/java/com/solace/connector/beam/examples/SolaceBigQuery.java) example receives a message from PubSub+ queue, transforms it, and then writes it to a BigQuery table. 
+
+1. Create a BigQuery table with appropriate schema.
+1. Modify SolaceBigQuery sample to add details about your table (project id, dataset id, and table id) in the following section:
+```java
+TableReference tableSpec =
+                new TableReference()
+                        .setProjectId("<project_id>")
+                        .setDatasetId("<dataset_id>")
+                        .setTableId("<table_id>");
+```
```

Comment:
Could we add the properties to the sample's `Options` interface?

For example:
- `bigqueryProject` for `<project_id>`
- `bigqueryDataset` for `<dataset_id>`
- `bigqueryTable` for `<table_id>`